### PR TITLE
update languages

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/Viva/issues/416
image

@ruthholls please review thanks

### CODE REVIEW

File: Viva/topics/plan-topic-experiences.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ You must be [subscribed to Viva Topics](https://www.microsoft.com/microsoft-viva
 All users who are going to use Topics require a **Topic Experiences** license. Assigning licenses is covered in [Set up Microsoft Viva Topics](set-up-topic-experiences.md). If you plan to assign licenses programmatically, be sure the users already have licenses for Microsoft 365 (for example, E3 or E5) before assigning the Topic Experiences license. 
 
 > [!Important] 
-> Topics will only crawl English content.
+> Topics crawl content in English, French, German and Spanish.
```
Comment (JohanFreelancer9) [inline Viva/topics/plan-topic-experiences.md:47]:
```suggestion
> Topics crawl content in English, French, German, and Spanish.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by ruthholls** [comment]
Looks good, thank you.

**Comment by get-itips** [comment]
@tiburd we have the ok of the article's author

**Pull request merged**